utils/index.d.ts

```diff
@@ -1526,6 +1515,17 @@ export declare abstract class AzureWizardPromptStep<T extends IActionContext> {
     public abstract shouldPrompt(wizardContext: T): boolean;
 }
 
+/**
+ * name: A displayable name of the step
+ * value: A displayable value chosen by the user (The label of the chosen value for the step)
+ * valueInContext: The name which can be used to access the value in the wizard context
+ */
+export type ConfirmationViewProperty = {
+    name: string;
+    value: string;
+    valueInContext: string;
+}
```

Comment:
nit: If you put the property descriptions on separate comments above each property in the object then the doc strings will show up in the hovers when people are using the interface.